src/components/FloatingBar/FloatingBar.css

```diff
@@ -8,12 +8,12 @@
   border: 1px solid var(--black-300);
   background: var(--white-900);
   box-shadow: 0 0 25px 0 var(--alpha-black-400);
-  padding: 12px 18px 12px 16px;
+  padding: 12px 1em 12px 1em;
```

Comment:
You could write this like `padding: 12px 1em` (or 1em 12px, don't remember)

Reply:
padding: 12px 1em